### PR TITLE
Create-react-app with TypeScript scripts

### PR DESCRIPTION
**Pull request by tbarlow12**

### CODE REVIEW

File: public/manifest.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
```
Comment (elizabethhalper) [inline public/manifest.json]: nit: I would change this to what we want it to be--VoTT React Library or something

File: src/components/tagsInput/tagsInput.tsx [hunk 2]
```diff
@@ -45,21 +56,21 @@ export interface ITagsInputProps {
...
+    onCtrlTagClick?: (tag: ITag) => void;
+    onShiftTagClick?: (tag: ITag) => void;
+    onCtrlShiftTagClick?: (tag: ITag) => void; 
 }
 
 /**
  * Current state of tags input component
  * @member tags - IReactTag[] - tags used in lower level component
  * @member currentTagColorIndex - rotates initial color to apply to tags
- * @member selectedTag - tag that was most recently double clicked,
```
Comment (elizabethhalper) [inline src/components/tagsInput/tagsInput.tsx]: I'd keep selectedTag documentation in here since we are keeping it

Comment (tbarlow12) [reply src/components/tagsInput/tagsInput.tsx]: Good call. I forgot I put that back.

Comment (tbarlow12) [reply src/components/tagsInput/tagsInput.tsx]: Actually I meant to take it out entirely. It won't be used internally in the component. We can discuss more about it later

**Pull request merged**